README.md

```diff
@@ -345,8 +166,21 @@ By default, Cloudfuse will log to syslog. The default settings will, in some
 cases, log relevant file paths to syslog. If this is sensitive information, turn
 off logging or set log-level to LOG_ERR.  
 
+## Third-Party Notices
+See [notices](./NOTICE) for third party license notices.
+
+Qt is licensed under the GNU Lesser General Public License version 3, which is available at https://doc.qt.io/qt-6/lgpl.html
+
+WinFSP is licensed under the GPLv3 license with a special exception for Free/Libre and Open Source Software, which is available at https://github.com/winfsp/winfsp/blob/master/License.txt
+
+## Attribution
+WinFsp - Windows File System Proxy, Copyright (C) Bill Zissimopoulos https://github.com/winfsp/winfsp
+
 ## License
-This project is licensed under MIT.
+The Cloudfuse project is licensed under MIT.
+
+## Frequently Asked Questions
+A list of FAQs can be found [here](https://github.com/Seagate/cloudfuse/wiki/Frequently-Asked-Questions)
 
 ## Contributing
 This project welcomes contributions and suggestions.

```

Comment:
Do we have a link to some instructions or guidelines for contributions or suggestions? The invitation feels perfunctory without something to go do or read next.

Reply:
We do not, I will add that in the backlog